examples/ingest/cmd/root.go

```diff
@@ -110,6 +111,17 @@ func initConfig() {
 	if err != nil {
 		er(fmt.Sprintf("failed to create IndyKite Ingest Client: %v", err))
 	}
+	retryClient, err = ingest.NewRetryClient(context.Background(),
+		&ingest.RetryPolicy{
+			MaxAttempts:       4,
+			InitialBackoff:    "1s", // proto duration format according to spec: https://protobuf.dev/programming-guides/proto3/#json
```

Comment:
Really, I don't like this:

1. You are referring to JSON format of Proto message. But this `RetryPolicy` is not Proto message. And it is not possible to Marshall or Unmarshal to JSON.
2. You are parsing it again and again. If client fails, you cannot just grab backoff, you need to parse this string to duration again. Requires extra CPU time for nothing.
3. You are referring to Proto spec, which requires format `1s` or `1.000001s`. But if I pass `30ms` which is not valid Proto duration, it will still work. Because time.Duration supports that format.
4. Ad 1&3. If you would implement JSON parsing. Time duration could generate time like `30ms` which is again not valid according to spec you pointed out.

Reply:
fine, I'll change it